rskj-core/src/main/java/co/rsk/peg/utils/BridgeEventLoggerImpl.java

```diff
@@ -143,6 +143,24 @@ public void logCommitFederation(Block executionBlock, Federation oldFederation,
         addLog(encodedTopics, encodedData);
     }
 
+    @Override
+    public void logCommitFederationFailure(Block executionBlock, Federation proposedFederation) {
+        CallTransaction.Function event = BridgeEvents.COMMIT_FEDERATION_FAILED.getEvent();
+
+        byte[][] encodedTopicsSerialized = event.encodeEventTopics();
```

Comment:
Do you think this is necessary? I see there's no topic for this event. Is this result an empty array? I couldn't test it by myself because of the Java 17 issue in my local environment.

Reply:
it encodes the tx hash. Just like in all the other tests :)

Reply:
The first topic of every event is the event signature encoded